src/SFA.DAS.Roatp.CourseManagement.Web/Models/ProviderCourseLocations/ProviderCourseLocationViewModel.cs

```diff
@@ -0,0 +1,46 @@
+﻿using SFA.DAS.Roatp.CourseManagement.Application.Constants;
+using SFA.DAS.Roatp.CourseManagement.Domain.ApiModels;
+
+namespace SFA.DAS.Roatp.CourseManagement.Web.Models.ProviderCourseLocations
+{
+
+    public class ProviderCourseLocationViewModel
+    {
+        public string LocationName { get; set; }
+        public LocationType LocationType { get; set; }
+        public bool? HasDayReleaseDeliveryOption { get; set; }
+        public bool? HasBlockReleaseDeliveryOption { get; set; }
+
+        public string RegionName { get; set; }
+
+        public CourseLocationDeliveryOption DeliveryOption()
+        {
+            if ((HasDayReleaseDeliveryOption.HasValue && HasDayReleaseDeliveryOption.Value) &&
+                (HasBlockReleaseDeliveryOption.HasValue && HasBlockReleaseDeliveryOption.Value))
+            {
+                return CourseLocationDeliveryOption.DayAndBlockRelease;
```

Comment:
Enums are not required here. Add a string field in the view model and populate that from the constants.

Reply:
enums removed and replaced with string text

Reply:
This is fine. But I think we will need to move this logic centrally so it becomes more reusable. don't have to do it now, but consider it when we come to next use of it.